# Define the goal
int32 goal_check_product
int32 goal_check_product_description
---
# Define the result
#int32 total_dishes_cleaned
int32 result_product_description
bool result_presence_of_product
---
# Define a feedback message
float32 percent_complete
